internal/utils/csi.go

```diff
@@ -366,6 +366,11 @@ var NodePluginCsiAddonsContainerPort = corev1.ContainerPort{
 	ContainerPort: int32(9071),
 }
 
+// CSI snapshot-metadata sidecar container port definition
+var SnapshotMetadataGRPCPort = corev1.ContainerPort{
```

Comment:
We only capitalize the first latter, even in case of abbreviations. 
```suggestion
var SnapshotMetadataGrpcPort = corev1.ContainerPort{
```